modules/EvseManager/energy_grid/energyImpl.cpp

```diff
@@ -425,81 +425,83 @@ void energyImpl::handle_enforce_limits(types::energy::EnforcedLimits& value) {
                     auto powersupply_capabilities = mod->get_powersupply_capabilities();
 
                     // tell car our new limits
-                    types::iso15118_charger::DC_EVSEMaximumLimits evseMaxLimits;
+                    types::iso15118_charger::DcEvseMaximumLimits evseMaxLimits;
```

Comment:
not sure if it makes sense to make this `evseMaxLimits` to snake case `evse_max_limits` as part of this PR as well

Reply:
Yes, that's right, I'll have a look if I can also find such variables

Reply:
Find some more. Hopefully all